MLS.Agent/CommandLine/CommandLineParser.cs

```diff
@@ -467,5 +465,12 @@ Command Verify()
                 return verifyCommand;
             }
         }
+
+        private static KernelBase PrepareKernel(KernelBase kernel)
+        {
+            kernel.UseExtendDirective();
```

Comment:
`UseExtendDirective` should be called at the `CompositeKernel` level.

Reply:
Updated.